### DISCUSSION TITLE
Maximum Recursion Depth Reached Error

### DISCUSSION THREAD

**Discussion by charan1319** [flatironinstitute/CaImAn#1201] [Q&A]
Hello,

I'm trying to run the fit function on my cnmf object with 3D data but am getting a maximum recursion depth error. Is this simply because of the length of the recording? Either way, does anyone have suggestions for the best solution to this problem? Also, I would like to check the locations of cells that CaImAn found by plotting them on top of a frame of my recording but have been having some trouble getting this to work -- how can I do this? 

Thank you!

Code: 
<img width="665" alt="image" src="https://github.com/flatironinstitute/CaImAn/assets/37819328/72615f7a-541d-4b94-b1d2-03139225adca">

Error: 
<img width="1121" alt="image" src="https://github.com/flatironinstitute/CaImAn/assets/37819328/fd941803-2938-43bf-acbe-16b7460dd1e1">

**Comment by kushalkolar** [comment]
That's weird, it looks like bokeh tried to throw a warning but I'm not sure why bokeh would've been called during cnmf.fit. @pgunn

**Comment by pgunn** [comment]
Let's talk about the likely software bug in #1204

**Comment by EricThomson** [comment]
Closing this interested parties can see #1204
